hw/ip/rom_ctrl/data/rom_ctrl.hjson

```diff
@@ -32,6 +32,15 @@
       randcount: "128",
       randtype:  "data"
     }
+
+    { name:      "SkipCheck",
```

Comment:
Does this actually need exposing in the hjson? Since it's just a temporary bodge, I was imagining we'd flip it in the code and (once everything is wired up) just delete it.

Reply:
ah i'm doing this now "purely" because i want synthesis to see it, but was hiding it from dv / verilator / fpga.
let me know though, i didn't look at the scramble PR super closely, once the kmac portion is in, i definitely agree this is all gone.

Reply:
Oh, I see! Yep, makes sense to me.